dastardcommander/trigger_config.py

```diff
@@ -14,23 +14,22 @@ class is new."""
 
     def __init__(self, parent=None):
         QtWidgets.QWidget.__init__(self, parent)
-        self.ui = Ui_Trigger()
-        self.ui.setupUi(self)
```

Comment:
Did you omit `setupUI` by mistake? It sounds important. I guess it's implicit in `PyQt5.uic.loadUi(...)`?